internal/storage/utils_test.go

```diff
@@ -20,11 +20,22 @@ import (
 	"github.com/eclipse-kanto/software-update/hawkbit"
 )
 
-type ah struct {
-	Hash     hawkbit.Hash
-	Protocol hawkbit.Protocol
+type artifactData struct {
+	hash     hawkbit.Hash
+	protocol hawkbit.Protocol
+	local    bool
+	copy     bool
 }
 
+var (
+	partialDownload = func(progress int) bool {
+		return progress > 0 && progress < 100
+	}
+	completeDownload = func(progress int) bool {
```

Comment:
It seems that this function is not used. There is a copy in `feature` package that is used by the test case.